src/libtools/application.cc

```diff
@@ -66,7 +66,6 @@ int Application::Exec(int argc, char **argv) {
     AddProgramOptions()("verbose1", "  be very loud and noisy");
     AddProgramOptions()("verbose2,v", "  be extremly loud and noisy");
     AddProgramOptions("Hidden")("man", "  output man-formatted manual pages");
```

Comment:
Is the `man` option used at all?

Reply:
I thought you use that for something.

Reply:
No, we use that new `votca_help2doc` script everywhere now.

Reply:
I thought that converts the man files to sth else.

Reply:
It only uses the output of `--help`.

Reply:
wow then we can remove quite a bit of stuff. :)